dataprofiler/profilers/profile_builder.py

```diff
@@ -1112,6 +1114,19 @@ def _save_helper(self, filepath: str | None, data_dict: dict) -> None:
         # Restore all data labelers
         self._restore_data_labelers(data_labelers)
 
+    def _json_save_helper(self, filepath: str) -> None:
+        """
+        Save profiler to disk.
+
+        :param filepath: Path of file to save to
+        :type filepath: String
+        :return: None
+        """
+        if filepath is None:
+            filepath = ""
```

Comment:
```suggestion
            if filepath is None:
              filepath = "profile-{}.json".format(
                  datetime.now().strftime("%d-%b-%Y-%H:%M:%S.%f")
              )
```

Reply:
this is sneaky because it probably silently runs but doesn't do what we actually want ... 💀

Reply:
good call, should be same as pkl technique.